server/remote/msgraph/call.go

```diff
@@ -36,29 +37,44 @@ func (c *client) Call(method, path string, in, out interface{}) (responseData []
 	}
 
 	var inBody io.Reader
+	var contentType string
```

Comment:
0/5 I like what you did, but I wonder if it'd simplify (and improve the functionality) of the code to have contentType passed in by the caller. If `.Call` gets used widely, we can always add a `.CallJSON` to simplify the caller-side code.